.github/workflows/release.yaml

```diff
@@ -205,6 +206,7 @@ jobs:
           gh release view ${{ needs.extract-version-details.outputs.aliased-version }} || \
             gh release create ${{ needs.extract-version-details.outputs.aliased-version }} \
             --latest=false \
+            --draft=${{ inputs.kind != 'production' }} \
```

Comment:
I think it's fine to have a always-updating public nightly? From my understanding, this command is only one once-per-aliased version anyways right (just to create the `unstable` branch? (so we have to delete the unstable tag for this to run again)

Reply:
> this command is only one once-per-aliased version

Correct, I'll tweak it though because I want it to be accurate